![Build Status](https://github.com/obophenotype/cell-ontology/workflows/CI/badge.svg)
[![DOI](https://zenodo.org/badge/13996/obophenotype/cell-ontology.svg)](https://zenodo.org/badge/latestdoi/13996/obophenotype/cell-ontology)


<img src="https://user-images.githubusercontent.com/112839/107354015-9375bb00-6ac5-11eb-9394-7318df343ec2.png" width=30% height=30%>


# Cell Ontology 


An ontology of cell types

This is the repository that contains the source for the cell ontology. Most users do not need to use this repo directly.

To browse the ontology, we recommend using OLS:
https://www.ebi.ac.uk/ols/ontologies/cl

For more details on CL see:
http://obofoundry.org/ontology/cl.html

Documentation for the Cell Ontology is also available at https://obophenotype.github.io/cell-ontology/.

Editors: see the README in [src/ontology](src/ontology)

Training materials from the 2020 CL Training Workshop are available at https://github.com/obophenotype/cell-ontology-training.


## Twice monthly calls
  
Third Wednesday of month, 8am PT/11 am ET (CL)  
Fourth Monday of month, 10am PT/1pm ET (CL & Uberon)  
[Agenda here.](https://docs.google.com/document/d/1XvMbNvr0FEsdqGhg79BYCYEHSqUxRHMcvhbGizEAht8/edit)  

## Cite

Diehl,A.D., Meehan,T.F., Bradford,Y.M., Brush,M.H., Dahdul,W.M., Dougall,D.S., He,Y., Osumi-Sutherland,D., Ruttenberg,A., Sarntivijai,S., et al. (2016) The Cell Ontology 2016: enhanced content, modularization, and ontology interoperability. J. Biomed. Semantics, 7, 44.

## Applications

CL is used in a number of applications including: 

**HuBMAP**

HuBMAP develops tools to create an open, global atlas of the human body at the cellular level. The Cell Ontology is used in annotating cell types in the tools developed.

HuBMAP Consortium (2019) The human body at cellular resolution: the NIH Human Biomolecular Atlas Program. Nature, 574, 187–192

**Human Cell Atlas (HCA)**

The Human Cell Atlas (HCA) is an international group of researchers using a combination of these new technologies to create cellular reference maps. The HCA use CL to annotated cells in their reference maps.

Regev,A., Teichmann,S.A., Lander,E.S., Amit,I., Benoist,C., Birney,E., Bodenmiller,B., Campbell,P., Carninci,P., Clatworthy,M., et al. (2017) The Human Cell Atlas. Elife, 6.

**Single Cell Expression Atlas**

The EBI single cell expression atlas is an extension to EBI expression atlas that displays gene expression in single cells. Cell types in the single cell expression atlas linked with terms from the Cell Ontology.

Papatheodorou,I., Moreno,P., Manning,J., Fuentes,A.M.-P., George,N., Fexova,S., Fonseca,N.A., Füllgrabe,A., Green,M., Huang,N., et al. (2020) Expression Atlas update: from tissues to single cells. Nucleic Acids Res., 48, D77–D83.

**BRAIN Initiative Cell Census Network (BICCN)/Brain Data Standards Ontology**

The BICCN created a high-resolution atlas of cell types in the primary motor based on single cell transcriptomics. These cell types are represented in the brain data standards ontology which anchors to cell types in the cell ontology.

Tan,S.Z.K., Kir,H., Aevermann,B., Gillespie,T., Hawrylycz,M., Lein,E., Matentzoglu,N., Miller,J., Mollenkopf,T.S., Mungall,C.J., et al. (2021) Brain Data Standards Ontology: A data-driven ontology of transcriptomically defined cell types in the primary motor cortex. bioRxiv, 10.1101/2021.10.10.463703.

**ENCODE**

The National Human Genome Research Institute (NHGRI) launched a public research consortium named ENCODE, the Encyclopedia Of DNA Elements, in September 2003, to carry out a project to identify all functional elements in the human genome sequence. The ENCODE DCC uses Uberon to annotate samples

Malladi, V. S., Erickson, D. T., Podduturi, N. R., Rowe, L. D., Chan,
E. T., Davidson, J. M., … Hong, E. L. (2015). Ontology application and
use at the ENCODE DCC. Database : The Journal of Biological Databases
and Curation, 2015, bav010–. [doi:10.1093/database/bav010](https://doi.org/doi:10.1093/database/bav010)

**FANTOMS**

FANTOM5 is using Uberon and CL to annotate samples allowing for transcriptome analyses with cell-type and tissue-level specificity.

Lizio, M., Harshbarger, J., Shimoji, H., Severin, J., Kasukawa, T.,
Sahin, S., … Kawaji, H. (2015). Gateways to the FANTOM5 promoter level
mammalian expression atlas. Genome Biology, 16(1),
22. [doi:10.1186/s13059-014-0560-6](https://doi.org/doi:10.1186/s13059-014-0560-6)

**LINCS**

Metadata Standard and Data Exchange Specifications
to Describe, Model, and Integrate Complex and Diverse High-Throughput
Screening Data from the Library of Integrated Network-based Cellular
Signatures
(LINCS) http://jbx.sagepub.com/content/early/2014/02/11/1087057114522514.full
